modules/javafx.graphics/src/main/java/com/sun/glass/ui/monocle/LinuxArch.java

```diff
@@ -28,16 +28,17 @@
 import java.security.AccessController;
 import java.security.PrivilegedAction;
 
-@SuppressWarnings("removal")
 public class LinuxArch {
 
     private static final int bits;
 
     static {
-        bits = AccessController.doPrivileged((PrivilegedAction<Integer>) () -> {
```

Comment:
Combine?

Reply:
Good idea. This allows the static block to be removed.